examples/operating-system-manager.yaml

```diff
@@ -1076,7 +1076,8 @@ spec:
     spec:
       serviceAccountName: operating-system-manager-webhook
       containers:
-        - image: quay.io/kubermatic/operating-system-manager:v1.0.0
+        # TODO: Update this to a semver tag before release.
+        - image: quay.io/kubermatic/operating-system-manager:ea2250874bbcea46ad956cbba8972be7ffdf9ce2
```

Comment:
how about releasing new OSM version instead?

Reply:
There are few more things that I'd like to add in OSM before the next release.